pkg/subctl/cmd/join.go

```diff
@@ -265,13 +265,14 @@ func joinSubmarinerCluster(subctlData *datafile.SubctlData) {
 	operatorImage, err := image.ForOperator(imageVersion, repository, imageOverrideArr)
 	utils.ExitOnError("Error overriding Operator Image", err)
 	err = submarinerop.Ensure(status, clientConfig, OperatorNamespace, operatorImage, operatorDebug)
-	status.End(cli.CheckForError(err))
+	status.EndWith(cli.CheckForError(err))
+	status.EndWith(err)
```

Comment:
This isn't needed.

```suggestion
```

Reply:
yes. Realised it after pushing the commit. I just pushed an edit.